proof/dleq.go

```diff
@@ -0,0 +1,163 @@
+package proof
+
+import (
+	"errors"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/hash"
+	"github.com/dedis/crypto/random"
+)
+
+// This package provides functionality to create and verify non-interactive
+// zero-knowledge (NIZK) proofs for the equality (EQ) of discrete logarithms (DL).
+
+// DLEQ resembles a NIZK dlog-equality proof. Allows to handle multiple proofs.
```

Comment:
english: resemble - have a similar appearance to or qualities in common with (someone or something); look or seem like.
Probably you wanted to write: assembles. Or even better:

```
Discrete logarithm equality (DLEQ) holds one or more non-interactive zero-knowledge (NIZK) proofs.
```

Also comment the elements of the structure.